test/README.md

```diff
@@ -0,0 +1,11 @@
+The "Dracut Developer Guidelines" page in the main documentation has a
+section that explains how to use these tests:
+https://dracut-ng.github.io/dracut-ng/developer/hacking.html#_testsuite
+
+The sources of that documentation are located in the `doc_site/` folder
```

Comment:
It's possible to link directly to other files/directories in the repository.

Reply:
Done.